liberbatch.py

```diff
@@ -1,19 +1,15 @@
 import os, meichecker, liberupdatev5
+PATH = "your path here"

```

Comment:
If you want the user to go in and change the path, that should be documented. I think a command line argument would be generally preferable.